src/modules/pos/validator/hooks/queries/useRewardsLocked.js

```diff
@@ -0,0 +1,35 @@
+import { POS_REWARDS_LOCKED } from 'src/const/queries';
+import {
+  API_VERSION,
+} from 'src/const/config';
+import { useCustomQuery } from 'src/modules/common/hooks';
+
+/**
+ * Creates a custom hook to fetch claimable rewards by address
+ *
+ * @param {object} configuration - the custom query configuration object
+ * @param {object} configuration.config - the query config
+ * @param {object} configuration.config.params - the query config params
+ * @param {string} configuration.config.params.address - account address
+ * @param {string} configuration.config.params.publicKey - publicKey
+ * @param {string} configuration.config.params.name - name
+ * @param {string} configuration.config.params.limit - limit
+ * @param {string} configuration.config.params.offset - offset
```

Comment:
@oskarleonard Possibly update the documentation to specify which params are optional

Reply:
@soroushm Maybe you know some "hack" for this 
<img width="629" alt="image" src="https://user-images.githubusercontent.com/8784876/209700949-0bfb4a69-1b48-4b21-95f3-f3110735bd4c.png">

For now i will make all of them optional

Reply:
yup they are optional